src/components/ContentSidebar/SkillsSidebar.js

```diff
@@ -38,12 +38,40 @@ class SkillsSidebar extends React.PureComponent<Props, State> {
         api.getMetadataAPI(false).getSkills(
             file,
             (cards: Array<SkillCard>) => {
+                this.updatePreviewTranscript(cards);
                 this.setState({ cards });
             },
             noop
         );
     }
 
+    /**
+     * Updates Preview with transcript data
+     *
+     * @private
+     * @param {Array<SkillCard>} cards - Skills cards
+     * @return {void}
+     */
+    updatePreviewTranscript = (cards: Array<SkillCard>) => {
+        const { getPreviewer } = this.props;
+        const preview = getPreviewer ? getPreviewer() : null;
+        const viewer = preview ? preview.viewer : null;
+        const transcriptCard = cards.find((card) => card.skill_card_type === 'transcript');
+        if (!transcriptCard || !preview) {
+            return;
+        }
+
+        if (!viewer || !viewer.isLoaded() || viewer.isDestroyed()) {
```

Comment:
If we have a utility getViewer function, this can be encapsulated in that. I think this code might be repeated in one more place.